app/services/foreman_discovery/host_converter.rb

```diff
@@ -1,9 +1,12 @@
+require 'foreman_discovery/lldp_neighbors'
```

Comment:
This should not be necessary. Rails does auto loading pretty well. Let's remove this.
You might want to move the library to app/services as well (instead of lib/foreman_discovery/).

Reply:
It drove me mad to figure out what to place where, as I faced quite some inconsistencies. Well, at least I haven't been able to really grasp the rules. No wonder Ruby projects fire an such an immense amount of stat-calls.

I placed this code in a separate class to make it easier to test in an isolated way. Just, I failed to run the tests, gave some rake test:foreman_discovery examples triggered from the foreman base directory a try, but nothing happened.

Reply:
@timogoebel: any helpful hint on how to kick the test run would be greatly appreciated!